kahuna/public/js/search/index.js

```diff
@@ -109,6 +115,7 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
                 // keep the shortcut keys here to stop overriding
                 return new Map([
                     ['metadataPanel', 'm'],
+                    ['collectionsPanel', 'c'],
```

Comment:
Only concern is C is "crop" on the image page, and recent talks were hinting at possibly bringing that elsewhere, on the results page perhaps. Would be nice to have a letter that we could reuse elsewhere even on the image page for instance.